schema/org.jdc.template.datasource.database.main.MainDatabase/1.json

```diff
@@ -2,11 +2,11 @@
   "formatVersion": 1,
   "database": {
     "version": 1,
-    "identityHash": "4c164342919e649eb6dbd7cc108bb039",
+    "identityHash": "71e94c403e42a5499b19f51b409d02f3",
```

Comment:
- Revert this file (this file needs to stay intact for version 1 of the database).
- Change the version of the database to 2
- Commit generated file 2.json

Reply:
@jeffdcamp I really struggled with this.  I expected that I could revert my branch to where it was when I first got it from you.  At that point I would have version 1.  Then I assumed that I could make my changes and it would build the 2.json with the proper migration.  I could never get this to work.  It would install and run fine but only after deleting the app.  I assumed that it was an issue with the state that I got my database into with changes I made without bumping the version.

Reply:
These json files are used for testing.... it allows us to create any version of the database at anytime, so I can test migration.  All migration work is done manually in the AppModule.kt addMigrations() code